server/cli/kv.go

```diff
@@ -122,14 +122,11 @@ func runPut(cmd *cobra.Command, args []string) {
 	if kvDB == nil {
 		return
 	}
-	err := kvDB.Tx(func(tx *client.Tx) error {
```

Comment:
I was taken aback that tobias' loop would involve transactions at all. I see the argument that this maybe should be all or nothing, but then again, why should you expect different behavior from a command line tool like this between `cockroach kv put a 1 b 2` and `cockroach kv put a 1 && cockroach kv put b 2`? The use of a transaction here wasn't documented consistently and just feels wrong considering we can't otherwise specify them through the command line.

Although it's easy to imagine ways you could find it useful, I really dislike the idea that people would start relying on things like "oh that creates an implicit transaction..." It's complexity where there's no reason to expect it either in the syntax or by convention.